server/data/data-library/src/main/java/org/apache/james/mailrepository/lib/AbstractMailRepository.java

```
@@ -135,12 +136,14 @@ public void remove(Collection<Mail> mails) throws MessagingException {
     }
 
     /**
+     * @throws MessagingException 
      * @see org.apache.james.mailrepository.api.MailRepository#remove(String)
      */
     public void remove(String key) throws MessagingException {
```

Why document an exception that cannot be raised (swallowing catch detected!) ?